test/cpp/doctest/DOCTest.cmake

```diff
@@ -2,7 +2,7 @@ include(FetchContent)
 
 FetchContent_Declare(doctest
                      GIT_REPOSITORY https://github.com/onqtam/doctest.git
-                     GIT_TAG 2.3.7)
+                     GIT_TAG 2.4.6)
```

Comment:
Which is the first version which supports `--no-breaks` option?
Why did you update?
I like to have the oldest supported version if it's not too inconvenient.

Reply:
The option has been in 2.3.7 already but did not work correctly, which makes it hard to test. Since there are no unit tests for this feature and doctest yet, we can go back to the old version I suppose.